docs/index.md

```diff
@@ -11,15 +11,40 @@ This is the documentation for the `cfgov-refresh` project, a redesign of the [ww
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
+- [Apache]() - Version? Which project exactly? https://apache.org/index.html#projects-list
```

Comment:
Perhaps @rosskarchner would be the person most easily able to answer this?

Reply:
It's also not quite right to include Apache in this list, as it's not a (current) part of normal cfgov-refresh development workflow.

Reply:
I would vote to exclude this from this documentation, as Apache isn't necessary/used at all as part of normal development on this repository. At such time as we move an Apache-based development option here (@wpears), we can add it.

Reply:
OK, I'll remove it from this, but Apache was specifically called out within the Doc Journey Map as something that is undocumented, so I'm adding a note to the project board to figure out what exactly if anything needs to be documented about our use of it.